### ISSUE TITLE
Feature request: Rocksmith support

### ISSUE THREAD

**Issue by ise73** [mdias/rs_asio#137]
Is there a chance to support the old classic Rocsmith?

**Comment by mdias** [comment]
This is not currently a planned feature.

**Comment by mdias** [comment]
@andryxa274 if that device has ASIO drivers it should work.
Keep in mind the `Audio Interfaces reported to work well` list is not a full list of devices that will work well with RS ASIO; it only reflects what users already reported to be working, but most devices with ASIO drivers and 48000hz support should work.

**Comment by andryxa274** [comment]
@mdias so valeton gp 200 support asio and I can see it in RS_ASIO-log.txt but it didn't work, valeton support only 44.1kHz( maybe that why, is it posible to do something with that?

**Comment by mdias** [comment]
> @mdias so valeton gp 200 support asio and I can see it in RS_ASIO-log.txt but it didn't work, valeton support only 44.1kHz( maybe that why, is it posible to do something with that?

Right, 44.1khz won't work. However I [RSMods](https://github.com/Lovrom8/RSMods) enables support alternative sample rates. It's worth a try.

**Comment by andryxa274** [comment]
> > @mdias so valeton gp 200 support asio and I can see it in RS_ASIO-log.txt but it didn't work, valeton support only 44.1kHz( maybe that why, is it posible to do something with that?
> 
> Right, 44.1khz won't work. However I [RSMods](https://github.com/Lovrom8/RSMods) enables support alternative sample rates. It's worth a try.

Thanks, RSmode help, kinde of, output work fine but input guitar didn't see(  I try from 0 to 6 chanel. Have no idea what to try

**Comment by mdias** [comment]
Please post the contents of RS_ASIO.ini and RS_ASIO-log.txt

**Comment by andryxa274** [comment]
@mdias thanks for help, is it ok to send it as .txt?
[RS_ASIO-log.txt](https://github.com/mdias/rs_asio/files/10869288/RS_ASIO-log.txt)
[RS_ASIO.ini.txt](https://github.com/mdias/rs_asio/files/10869297/RS_ASIO.ini.txt)

**Comment by mdias** [comment]
We can see in the log a bunch of:
> 27.363 [INFO]  {ASIO IN 0} DebugWrapperAudioClient<struct IAudioClient3>::IsFormatSupported - ShareMode: Exclusive
> 27.363 [ERROR]    unsupported sample rate: 48000

Unfortunately it seems that the game is still insisting on 48khz for the input signal, so unless RSMods has support for 44.1kHz on the inputs too, this might not work.
